src/Ethereum/ABI/Bytes.h

```diff
@@ -12,70 +12,87 @@
 
 namespace TW::Ethereum {
 
-// dynamic bytes
+/// Compute size when padded to 32-byte boundary; round up to multiple of 32.
+inline size_t paddedTo32(size_t len) { return ((len + 31) / 32) * 32; }
 
-bool is_dynamic(Data) {
+/// Compute padding needed to next 32-byte boundary.
+inline size_t padNeeded32(size_t len) { return ((len + 31) / 32) * 32 - len; }
```

Comment:
looks like we can add a `bool` parameter to  `paddedTo32` and merge these two functions?

Reply:
I think that would not make it more readable, I would not do.